src/com/amazon/ion/benchmark/WriteRandomIonValues.java

```diff
@@ -381,13 +416,19 @@ public static Timestamp writeTimestamp(Timestamp.Precision precision,  Timestamp
      * @param type determines which type of data will be generated [blob | clob].
      * @throws Exception if an error occurs when building up the writer.
     */
-    public static void writeRandomLobs(int size, IonType type, String format, String path) throws Exception {
+    public static void writeRandomLobs(int size, IonType type, String format, String path, IonStruct constraintStruct) throws Exception {
```

Comment:
add a doc comment for parameter `constraintStruct`. Also, the doc comments for the parameters can be ordered as they appear in the function call. 
```
     /** 
     * ...
     * @param size specifies the size in bytes of the generated file.
     * @param type determines which type of data will be generated [blob | clob].
     * @param format ...
     * @param path ...
     * @param constraintStruct ...
     */
```